docs/en/operation/User-CLI.md

```diff
@@ -1302,6 +1302,33 @@ $ ./bin/alluxio fs rm /tmp/unused-file
 $ ./bin/alluxio fs rm --alluxioOnly /tmp/unused-file2
 ```
 
+When deleting only from Alluxio but leaving the files in UFS, you can use `-U` and `-Dalluxio.user.file.metadata.sync.interval=-1`
+to skip the metadata sync and the UFS check. This will save time and memory consumption on the Alluxio master.
+```console
+$ bin/alluxio fs rm -R -U --alluxioOnly -Dalluxio.user.file.metadata.sync.interval=-1 /path
+```
+
+When deleting a large directory (with millions of files) recursively both from Alluxio and UFS,
+the operation is expensive. 
+
+We recommend doing the deletion in the following way:
+1. Using the corresponding tool to check the UFS path directly and make sure everything can be deleted safely. 
```

Comment:
By "check" i mean that sanity check you would do before deleting a large dir in your own PC. It's probably not possible(or necessary) to check every file or subdir, but at least something can be done easily like looking at the top level, take a look, if anything looks interesting, step into that dir and take deeper look, and iterate until you feel confident.

At this part of doc I'd like to mention that `-U` does not really save your life, despite the fact that it's called a "check". All this `-U` does is to skip the files where the in-Alluxio and in-UFS views are different. This does not identify files you don't want to remove, nobody else can do that.

Also a metadata sync will not make the situation any better. Because knowing Alluxio is in sync with the UFS does not make anything better. You will still have to do the sanity check.

All I'm trying to add are mainly:
1. The UFS check (by not `-U`) is not your friend, neither is metadata sync.
2. You should really do the sanity check yourself.